pkg/resource/detector/detector.go

```diff
@@ -0,0 +1,101 @@
+package detector
+
+import (
+	runtime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/discovery"
+	"time"
+)
```

Comment:
I believe something in the Go tooling (gofmt? I can't remember all the names) would change this to the following (standard library packages, empty line, other things):

```go
import (
	"time"

	runtime "k8s.io/apimachinery/pkg/runtime"
	"k8s.io/apimachinery/pkg/runtime/schema"
	"k8s.io/client-go/discovery"
}
```

It's fine the way it is, but it might be worth configuring your editor/IDE to organise the imports on file save.